docker/Dockerfiles/docker-mx-ng-install.sh

```diff
@@ -59,10 +64,10 @@ docker_mx_dir="/home/dockuser/ng-mx"
 
 script='run-ng-mx-install.sh'
 
-docker run --rm \
+nvidia-docker run --rm \
```

Comment:
Why this change?

Reply:
Because I pull the docker images from nvidia/cuda:9.1-cudnn7-devel for testing Nvidia GPU.

Reply:
I think this change makes the CPU only jobs also use the nvidia docker image? Not sure that's a problem.

Reply:
Good catch!